spec/README.mediawiki

```diff
@@ -1130,30 +1152,99 @@ DEX will send a list of '''Match objects'''.
 |-
 | rate      || int    || the rate that the order matched at (as quote asset per unit base asset, atoms)
 |-
-| address   || string || the client's receiving address
+| address   || string || the counterparty's receiving address
 |-
-| timestamp || int    || UTC timestamp. order broadcast time
+| sig       || string || DEX's hex-encoded signature of the serialized notification data. serialization described below
+|}
+
+'''Match notification serialization'''
+
+{|
+! field      !! size (bytes)  !! description
+|-
+| orderid    || 32 || the order ID
+|-
+| matchid    || 8  || the ID assigned to this match
+|-
+| quantity   || 8  || the matched amount, in atoms of the base asset
+|-
+| rate       || 8  || the rate that the order matched at
+|-
+| timestamp  || 8  || the client's UNIX timestamp
+|-
+| address    || varies || ASCII encoded receiving address for the match
+|}
+
+The client will respond with a list of signed match acknowledgements.
+
+After a client broadcasts their initialization transaction, they are
+expected to report the transaction details to the server for verification and
+relay to the matching party.
+
+'''JSON-RPC method:''' <code>init</code>, '''originator:''' client
+
+{|
+! field      !! type   !! description
+|-
+| orderid    || 32 || the order ID
+|-
+| matchid    || int    || the matchid, retrieved from the [[#Match_notifications|match notification]]
+|-
+| txid       || string || the hex-encoded transaction ID
+|-
+| vout       || int    || the output index
+|-
+| timestamp  || int    || UTC timestamp. order broadcast time
```

Comment:
UTC -> UNIX for consistency.  Could even make it UNIX epoch time to be even more explicit

Reply:
The word *epoch* has another meaning in the spec. I wouldn't want to confuse anyone. Should be UNIX though.